### DISCUSSION TITLE
How can i add it on my warp for linux ?

### DISCUSSION THREAD

**Discussion by ind4skylivey** [starship/starship#5791] [Q&A]
i try the command say on the Starship site 

`starship preset tokyo-night -o ~/.config/starship.toml`
 
 but nothing happen on my terminal warp, i mean nothing changes .
 
 anyone know how do to it ?

**Comment by davidkna** [comment]
Warp uses its own prompt by default. You may need to enable support for custom prompts in the warp settings.

**Reply by ind4skylivey** [reply]
ok great! and where can i find that option D: ?

**Reply by davidkna** [reply]
Try looking at https://docs.warp.dev/appearance/prompt

**Comment by ind4skylivey** [comment]
thank you! i am goona try my best ^^

**Comment by ind4skylivey** [comment]
![Screenshot_20240223_161029](https://github.com/starship/starship/assets/98955416/b6ec4a57-96d3-4e9e-ab90-427f8048936f)

It Works!!
